docs/source/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+PAPER         =
+BUILDDIR      = _build
```

Comment:
is this the canonical sphinx build dir?  what gets stored here?  maybe we should choose a more descriptive name?